test/ComposerRequireCheckerTest/FileLocator/LocateComposerPackageDirectDependenciesSourceFilesTest.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+namespace ComposerRequireCheckerTest\FileLocator;
+
+use ComposerRequireChecker\FileLocator\LocateComposerPackageDirectDependenciesSourceFiles;
+use org\bovigo\vfs\vfsStream;
+use org\bovigo\vfs\vfsStreamDirectory;
+use PHPUnit\Framework\TestCase;
+
+class LocateComposerPackageDirectDependenciesSourceFilesTest extends TestCase
```

Comment:
Please add `/** @covers \ComposerRequireChecker\FileLocator\LocateComposerPackageDirectDependenciesSourceFiles */` to prevent coverage leaks